File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/ProfilePictureUpdaterDTO.java
package com.example.myapplication.dtos;

/**
 * DTO object used to update user's profile picture.
 * When sending to the WCF web service, picture is stored as a base64 String object.
 */
public class ProfilePictureUpdaterDTO {
    private int UserId;
    private String Picture;

    public ProfilePictureUpdaterDTO(int userId, String picture) {
        UserId = userId;
        Picture = picture;
    }
}

File: Project/WebService/Services/DTOs/ProfilePictureUpdaterDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="ProfilePictureUpdaterDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The profile picture updater dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace Services.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The profile picture updater dto.
    /// </summary>
    [DataContract]
    public class ProfilePictureUpdaterDTO
    {
        /// <summary>
        /// Gets or sets the user id.
        /// </summary>
        [DataMember]
        public int UserId { get; set; }

        /// <summary>
        /// Gets or sets the picture.
        /// </summary>
        [DataMember]
        public string Picture { get; set; }
    }
}